Sources/SourceKitLSP/Clang/ClangLanguageServer.swift

```diff
@@ -490,6 +490,13 @@ extension ClangLanguageServerShim {
     forwardRequestToClangdOnQueue(req)
   }
 
+  func inlayHints(_ req: Request<InlayHintsRequest>) {
+    // Currently a Swift-specific, non-standard request.
```

Comment:
clangd has its own inlineHints extension that I would assume matches the proposal fairly exactly (since it's a clangd developer who wrote the proposal).  Should we forward this request as `clangd/inlayHints`?

Reply:
Yes that would be nice, `clangd` currently implements a slightly-older version of the proposal though that isn't strictly compatible from what I can tell: https://github.com/llvm/llvm-project/blob/316da54/clang-tools-extra/clangd/Protocol.h#L1488-L1527

Therefore I wasn't sure whether temporarily adding a custom request for `clangd` would be worth the effort currently. WDYT?